test/xpu_api/utilities/optional/optional.object/optional.object.swap/swap_optional_object_swap.pass.cpp

```diff
@@ -0,0 +1,239 @@
+// -*- C++ -*-
```

Comment:
The name of the test is confusing because swap is duplicated. I would simply name it `member_swap_optional.pass.cpp`

Reply:
Fixed